lacework/resource_lacework_alert_channel_gcp_pub_sub.go

```diff
@@ -0,0 +1,260 @@
+package lacework
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/lacework/go-sdk/api"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceLaceworkAlertChannelGcpPubSub() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLaceworkAlertChannelGcpPubSubCreate,
+		Read:   resourceLaceworkAlertChannelGcpPubSubRead,
+		Update: resourceLaceworkAlertChannelGcpPubSubUpdate,
+		Delete: resourceLaceworkAlertChannelGcpPubSubDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkIntegration,
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"intg_guid": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"created_or_updated_time": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"topic_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"credentials": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"client_id": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"client_email": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"private_key": {
+							Type:     schema.TypeString,
+							Required: true,
```

Comment:
This element **must** be marked as sensitive. Please check how this was implemented here https://github.com/lacework/terraform-provider-lacework/blob/master/lacework/resource_lacework_integration_gcp_cfg.go#L56-L73